### ISSUE TITLE
maven build error

### ISSUE THREAD

**Issue by hibira** [aws-observability/application-signals-demo#46]
`$. An error occurs in /mvnw clean install -P buildDocker.
This is because there aren't enough mixin definitions when compiling less.
Below is the error portion of the build log.

```
[INFO] /home/ubuntu/environment/workshopfiles/application-signals-demo/spring-petclinic-api-gateway/src/main/less
[INFO] Executing the mojo:
[INFO] Wro4j Model path: /home/ubuntu/environment/workshopfiles/application-signals-demo/spring-petclinic-api-gateway/src/main/wro/wro.xml
[INFO] targetGroups: null
[INFO] minimize: true
[INFO] ignoreMissingResources: null
[INFO] parallelProcessing: false
[INFO] buildDirectory: /home/ubuntu/environment/workshopfiles/application-signals-demo/spring-petclinic-api-gateway/target
[INFO] destinationFolder: /home/ubuntu/environment/workshopfiles/application-signals-demo/spring-petclinic-api-gateway/target
[INFO] cssDestinationFolder: /home/ubuntu/environment/workshopfiles/application-signals-demo/spring-petclinic-api-gateway/target/classes/static/css
[INFO] The following groups will be processed: [petclinic]
[INFO] folder: /home/ubuntu/environment/workshopfiles/application-signals-demo/spring-petclinic-api-gateway/target/classes/static/css
[INFO] processing group: petclinic.css
[ERROR] Failed to compile less resource: ro.isdc.wro.model.resource.Resource@6cd61804[CSS,petclinic,true].
[ERROR] 347:3 Could not find mixin named ".label-success"..
[ERROR] 351:3 Could not find mixin named ".label-warning"..
[ERROR] 355:3 Could not find mixin named ".label-danger"..
[ERROR] 359:3 Could not find mixin named ".label-info"..
[ERROR] 374:23 The variable "@alert-success-bg" was not declared..
[ERROR] 374:23 Wrong argument type to function 'fade', expected color saw faulty expression..
[ERROR] 374:48 The variable "@alert-success-border" was not declared..
[ERROR] 374:71 The variable "@alert-success-text" was not declared..
[ERROR] 374:3 Could not find mixin named ".alert-variant"..
[ERROR] 377:23 The variable "@alert-info-bg" was not declared..
[ERROR] 377:23 Wrong argument type to function 'fade', expected color saw faulty expression..
[ERROR] 377:45 The variable "@alert-info-border" was not declared..
[ERROR] 377:65 The variable "@alert-info-text" was not declared..
[ERROR] 377:3 Could not find mixin named ".alert-variant"..
[ERROR] 380:23 The variable "@alert-warning-bg" was not declared..
[ERROR] 380:23 Wrong argument type to function 'fade', expected color saw faulty expression..
[ERROR] 380:48 The variable "@alert-warning-border" was not declared..
[ERROR] 380:71 The variable "@alert-warning-text" was not declared..
[ERROR] 380:3 Could not find mixin named ".alert-variant"..
[ERROR] 383:23 The variable "@alert-danger-bg" was not declared..
[ERROR] 383:23 Wrong argument type to function 'fade', expected color saw faulty expression..
[ERROR] 383:47 The variable "@alert-danger-border" was not declared..
[ERROR] 383:69 The variable "@alert-danger-text" was not declared..
[ERROR] 383:3 Could not find mixin named ".alert-variant"..
[ERROR] Failed to process the resource: ro.isdc.wro.model.resource.Resource@6cd61804[CSS,petclinic,true] using processor: ro.isdc.wro.extensions.processor.css.Less4jProcessor@56a24f0. Reason: Could not compile less. 24 error(s) occurred:
ERROR 347:3 Could not find mixin named ".label-success".
 346: .deployment-status-deployed {
 347:   .label-success;
 348: }

ERROR 351:3 Could not find mixin named ".label-warning".
 350: .deployment-status-incomplete {
 351:   .label-warning;
 352: }

...

[ERROR] Exception occured while processing: ro.isdc.wro.WroRuntimeException: Could not compile less. 24 error(s) occurred:
ERROR 347:3 Could not find mixin named ".label-success".
 346: .deployment-status-deployed {
 347:   .label-success;
 348: }

ERROR 351:3 Could not find mixin named ".label-warning".
 350: .deployment-status-incomplete {
 351:   .label-warning;
 352: }

...
, class: ro.isdc.wro.WroRuntimeException,caused by: com.github.sommeri.less4j.Less4jException
ro.isdc.wro.WroRuntimeException: Could not compile less. 24 error(s) occurred:
ERROR 347:3 Could not find mixin named ".label-success".
 346: .deployment-status-deployed {
 347:   .label-success;
 348: }

ERROR 351:3 Could not find mixin named ".label-warning".
 350: .deployment-status-incomplete {
 351:   .label-warning;
 352: }

...

    at ro.isdc.wro.WroRuntimeException.wrap (WroRuntimeException.java:69)
    at ro.isdc.wro.WroRuntimeException.wrap (WroRuntimeException.java:54)
    at ro.isdc.wro.extensions.processor.css.Less4jProcessor.process (Less4jProcessor.java:106)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.LazyProcessorDecorator.process (LazyProcessorDecorator.java:49)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.SupportAwareProcessorDecorator.process (SupportAwareProcessorDecorator.java:39)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.ExceptionHandlingProcessorDecorator.process (ExceptionHandlingProcessorDecorator.java:56)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.BenchmarkProcessorDecorator.process (BenchmarkProcessorDecorator.java:44)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.DefaultProcessorDecorator.process (DefaultProcessorDecorator.java:42)
    at ro.isdc.wro.model.group.processor.GroupsProcessor$1.process (GroupsProcessor.java:134)
    at ro.isdc.wro.model.group.processor.GroupsProcessor.applyPostProcessors (GroupsProcessor.java:115)
    at ro.isdc.wro.model.group.processor.GroupsProcessor.process (GroupsProcessor.java:84)
    at ro.isdc.wro.cache.support.DefaultSynchronizedCacheStrategyDecorator.loadValue (DefaultSynchronizedCacheStrategyDecorator.java:101)
    at ro.isdc.wro.cache.support.DefaultSynchronizedCacheStrategyDecorator.loadValue (DefaultSynchronizedCacheStrategyDecorator.java:35)
    at ro.isdc.wro.cache.support.AbstractSynchronizedCacheStrategyDecorator.get (AbstractSynchronizedCacheStrategyDecorator.java:55)
    at ro.isdc.wro.manager.ResourceBundleProcessor.serveProcessedBundle (ResourceBundleProcessor.java:63)
    at ro.isdc.wro.manager.WroManager.process (WroManager.java:159)
    at ro.isdc.wro.maven.plugin.Wro4jMojo.processGroup (Wro4jMojo.java:277)
    at ro.isdc.wro.maven.plugin.Wro4jMojo.doExecute (Wro4jMojo.java:161)
    at ro.isdc.wro.maven.plugin.AbstractWro4jMojo.execute (AbstractWro4jMojo.java:169)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
Caused by: com.github.sommeri.less4j.Less4jException: Could not compile less. 24 error(s) occurred:
ERROR 347:3 Could not find mixin named ".label-success".
 346: .deployment-status-deployed {
 347:   .label-success;
 348: }

ERROR 351:3 Could not find mixin named ".label-warning".
 350: .deployment-status-incomplete {
 351:   .label-warning;
 352: }

...

    at com.github.sommeri.less4j.core.ThreadUnsafeLessCompiler.compile (ThreadUnsafeLessCompiler.java:87)
    at com.github.sommeri.less4j.core.ThreadUnsafeLessCompiler.compile (ThreadUnsafeLessCompiler.java:74)
    at com.github.sommeri.less4j.core.DefaultLessCompiler.compile (DefaultLessCompiler.java:44)
    at ro.isdc.wro.extensions.processor.css.Less4jProcessor.process (Less4jProcessor.java:98)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.LazyProcessorDecorator.process (LazyProcessorDecorator.java:49)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.SupportAwareProcessorDecorator.process (SupportAwareProcessorDecorator.java:39)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.ExceptionHandlingProcessorDecorator.process (ExceptionHandlingProcessorDecorator.java:56)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.BenchmarkProcessorDecorator.process (BenchmarkProcessorDecorator.java:44)
    at ro.isdc.wro.model.resource.processor.decorator.ProcessorDecorator.process (ProcessorDecorator.java:86)
    at ro.isdc.wro.model.resource.processor.decorator.DefaultProcessorDecorator.process (DefaultProcessorDecorator.java:42)
    at ro.isdc.wro.model.group.processor.GroupsProcessor$1.process (GroupsProcessor.java:134)
    at ro.isdc.wro.model.group.processor.GroupsProcessor.applyPostProcessors (GroupsProcessor.java:115)
    at ro.isdc.wro.model.group.processor.GroupsProcessor.process (GroupsProcessor.java:84)
    at ro.isdc.wro.cache.support.DefaultSynchronizedCacheStrategyDecorator.loadValue (DefaultSynchronizedCacheStrategyDecorator.java:101)
    at ro.isdc.wro.cache.support.DefaultSynchronizedCacheStrategyDecorator.loadValue (DefaultSynchronizedCacheStrategyDecorator.java:35)
    at ro.isdc.wro.cache.support.AbstractSynchronizedCacheStrategyDecorator.get (AbstractSynchronizedCacheStrategyDecorator.java:55)
    at ro.isdc.wro.manager.ResourceBundleProcessor.serveProcessedBundle (ResourceBundleProcessor.java:63)
    at ro.isdc.wro.manager.WroManager.process (WroManager.java:159)
    at ro.isdc.wro.maven.plugin.Wro4jMojo.processGroup (Wro4jMojo.java:277)
    at ro.isdc.wro.maven.plugin.Wro4jMojo.doExecute (Wro4jMojo.java:161)
    at ro.isdc.wro.maven.plugin.AbstractWro4jMojo.execute (AbstractWro4jMojo.java:169)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:954)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] spring-petclinic-microservices 2.6.7 ............... SUCCESS [  2.354 s]
[INFO] spring-petclinic-admin-server ...................... SUCCESS [02:10 min]
[INFO] spring-petclinic-customers-service ................. SUCCESS [ 53.428 s]
[INFO] spring-petclinic-vets-service ...................... SUCCESS [ 45.695 s]
[INFO] spring-petclinic-visits-service .................... SUCCESS [ 38.828 s]
[INFO] spring-petclinic-config-server ..................... SUCCESS [ 34.170 s]
[INFO] spring-petclinic-discovery-server .................. SUCCESS [ 44.801 s]
[INFO] spring-petclinic-api-gateway 2.6.7 ................. FAILURE [ 58.642 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 06:50 min
[INFO] Finished at: 2024-08-15T05:44:22Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal ro.isdc.wro4j:wro4j-maven-plugin:1.8.0:run (default) on project spring-petclinic-api-gateway: Exception occured while processing: ro.isdc.wro.WroRuntimeException: Could not compile less. 24 error(s) occurred:
[ERROR] ERROR 347:3 Could not find mixin named ".label-success".
[ERROR]  346: .deployment-status-deployed {
[ERROR]  347:   .label-success;
[ERROR]  348: }
[ERROR] 
[ERROR] ERROR 351:3 Could not find mixin named ".label-warning".
[ERROR]  350: .deployment-status-incomplete {
[ERROR]  351:   .label-warning;
[ERROR]  352: }
[ERROR] 
[ERROR] ...
[ERROR] , class: ro.isdc.wro.WroRuntimeException,caused by: com.github.sommeri.less4j.Less4jException
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :spring-petclinic-api-gateway
```

**Comment by hibira** [comment]
https://github.com/aws-observability/application-signals-demo/pull/47

**Comment by hibira** [comment]
It is caused by the upgrade of the pom.xml bootstrap version
```
 <webjars-bootstrap.version>3.3.7-1</webjars-bootstrap.version>
```
The error was resolved by reverting to

**Comment by majanjua-amzn** [comment]
Thanks for pointing that out! We fixed it in #52
